charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -173,6 +165,30 @@ data:
             ${FILTER3}
         </Location>
 
+        # DRS GetServiceInfo endpoint can be accessed without any authentication.
+        # TODO: do we benefit from a more aggressive regex here?
+        <Location ${PROXY_PATH3}/drs/v1/service-info>
+            ${AUTH_TYPE}
+            ${AUTH_REQUIRE}
+        </Location>
+
+        # DRS OptionsObject endpoint can be accessed without any authentication.
+        # DRS PostObject endpoint will use Passport auth, so the user does not need to pass through B2C login.
+        <LocationMatch "^${PROXY_PATH3}/drs/v1/objects/([^\/]+)$">
```

Comment:
Not convinced of the regex.  Trying not to match any paths nested within `…/drs/v1/objects/`.